src/CMakeLists.txt

```diff
@@ -137,7 +137,7 @@ install(EXPORT ${LIB_EXPORT_NAME}
 
 # Intel MKL-DNN compat cmake files
```

Comment:
(random place) please change the commit messages -- start them with `build: ` instead of `src/CMakefile.txt`